Makefile

```diff
@@ -4,12 +4,9 @@ TESTDIR=tmp-test-dir-with-unique-name
 PYTEST_COV_ARGS=--cov=$(PROJECT) --cov-config=../pyproject.toml \
```

Comment:
Seems like pytest-cov also support `pyproject.toml` config (see https://coverage.readthedocs.io/en/5.5/changes.html#version-5-0b1-2019-11-11). But let's open another PR for this to keep this one small :slightly_smiling_face: